frontend/src/e2e-playwright/specs/5_employee/vasu.spec.ts

```diff
@@ -86,70 +85,113 @@ describe('Vasu document page', () => {
   beforeEach(async () => {
     page = await Page.open()
     await employeeLogin(page, 'ADMIN')
-    vasuPage = await openDocument()
-  })
-
-  test('An unpublished vasu document has no followup questions', async () => {
-    await waitUntilEqual(() => vasuPage.followupQuestionCount, 0)
   })
 
-  describe('With a finalized document', () => {
-    const finalizeDocument = async () => {
-      await vasuPage.finalizeButton.click()
-      await vasuPage.modalOkButton.click()
-      await vasuPage.modalOkButton.click()
-      vasuPage = await openDocument()
-      await vasuPage.assertDocumentVisible()
-    }
-
-    beforeAll(async () => {
-      page = await Page.open()
-      await employeeLogin(page, 'ADMIN')
-      vasuPage = await openDocument()
-      await finalizeDocument()
-    })
-
-    test('A published vasu document has one followup question', async () => {
-      await waitUntilEqual(() => vasuPage.followupQuestionCount, 1)
-    })
-
-    test('Adding a followup comment renders it on the page', async () => {
-      vasuPage = await editDocument()
-      await vasuPage.inputFollowupComment('This is a followup')
-      await waitUntilEqual(
-        () => vasuPage.followupEntryTexts,
-        ['This is a followup']
+  describe('Fill out document', () => {
+    test('Fill the authors section', async () => {
+      const vasuEditPage = await editDocument()
+      const authors = vasuEditPage.authorsSection
+      await authors.primaryFirstNameInput.fill('Leena')
+      await authors.primaryLastNameInput.fill('Virtanen')
+      await authors.primaryTitleInput.fill('Johtaja')
+      await authors.primaryPhoneNumberInput.fill('1234565')
+
+      expect(await authors.otherFieldsCount).toBe(1)
+      await authors.otherFirstNameInput(0).fill('Veena')
+      await authors.otherLastNameInput(0).fill('Lirtanen')
+      await authors.otherTitleInput(0).fill('Hoitaja')
+      await authors.otherPhoneNumberInput(0).fill('1234565')
+
+      expect(await authors.otherFieldsCount).toBe(2)
+      await authors.otherFirstNameInput(1).fill('Aneev')
+      await authors.otherLastNameInput(1).fill('Nenatril')
+      await authors.otherTitleInput(1).fill('Hoitaja')
+      await authors.otherPhoneNumberInput(1).fill('5654321')
+
+      expect(await authors.otherFieldsCount).toBe(3)
+      await vasuEditPage.waitUntilSaved()
+      expect(await vasuEditPage.previewBtn.disabled).toBeFalsy()
```

Comment:
```
await waitUntilFalse(() => vasuEditPage.previewBtn.disabled)
```
Actually, in a case like this it would be slightly better to wrap this to a `vasuEditPage.assertEditDisabled()` function which calls `waitUntilFalse` under the hood.